bin/dev/install_tools.sh

```diff
@@ -21,7 +21,7 @@ mkdir -p "${SCF_BIN_DIR}"
 export SCF_BIN_DIR="$(unset CDPATH ; cd "${SCF_BIN_DIR}" && pwd)"
 
 echo "Fetching fissile $fissile_url ..."
-wget -q "$fissile_url"   -O - | tar xz --to-stdout fissile > "${FISSILE_BINARY}"
+wget -q "$fissile_url"   -O - | tar xz --to-stdout fissile > "${FISSILE_BINARY}.binary"
```

Comment:
Only if you're going to change this anyway: I think I've seen `foo.real` more often than `foo.binary`.